src/names.jl

```diff
@@ -18,6 +18,9 @@ NamedDims.dim(A::KaNda{L}, name) where {L} = NamedDims.dim(L, name)
 Base.axes(A::KaNda{L}, s::Symbol) where {L} = axes(A, NamedDims.dim(L,s))
 Base.size(A::KaNda{L,T,N}, s::Symbol) where {T,N,L} = size(A, NamedDims.dim(L,s))
 
+NamedDims.rename(A::KaNda, pairs::Pair{Symbol, Symbol}...) = KeyedArray(rename(parent(A), pairs...), axiskeys(A))
+NamedDims.rename(A::KaNda, names) = KeyedArray(rename(parent(A), names), axiskeys(A))
```

Comment:
Maybe simpler to accept whatever `rename` does?
```suggestion
NamedDims.rename(A::KaNda, names...) = KeyedArray(rename(parent(A), names...), axiskeys(A))
```
Otherwise looks fine, thanks!

Reply:
Oh yeah, I forgot how splatting worked : 😹.
I thought it would cause problems for tuples